core/drand_daemon.go

```diff
@@ -248,18 +248,25 @@ func (dd *DrandDaemon) LoadBeaconsFromDisk(metricsFlag string) error {
 		return err
 	}
 
+	groupHandlers := make([]metrics.GroupHandlers, 0, len(stores))
+
 	for beaconID, fs := range stores {
 		bp, err := dd.LoadBeaconFromStore(beaconID, fs)
 		if err != nil {
 			return err
 		}
 
-		// Start metrics server
 		if metricsFlag != "" {
-			_ = metrics.Start(metricsFlag, pprof.WithProfile(), bp.PeerMetrics)
+			log.DefaultLogger().Infow("", "metrics", "adding handler", "beacon", bp.beaconID)
```

Comment:
Please us the `bp.log` logger, it already includes the bp.beaconID, btw.

Reply:
👍🏻